db/seeds/development/1_people.rb

```diff
@@ -46,5 +46,5 @@ def amount(role_type)
 
 root = Group.root
 devs.each do |name, email|
-  seeder.seed_developer(name, email, root, Group::Verein::Leitung)
+  seeder.seed_developer(name, email, root, Group::Mitglieder::Mitglied)
```

Comment:
Ich fände es praktischer, wenn Entwickler etwas mächtigere Accounts bekommen. Daher mein Vorschlag:

```suggestion
  seeder.seed_developer(name, email, root, Group::Geschaeftsleitung::Mitarbeitende)
```